crypto/multisig/compact_bit_array.go

```diff
@@ -71,6 +71,21 @@ func (bA *CompactBitArray) SetIndex(i int, v bool) bool {
 	return true
 }
 
+// NumOfTrueBitsBefore returns the location of the given index, among the
```

Comment:
"returns the location of the given index" is not the right wording since it doesn't return a location, but number of bits true.